app/views/admin/recommended_tags/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%h3
```

Comment:
will it be clear here that this is a recommended tag and not a tag itself?

Reply:
Maybe, but I updated it regardless

![Screenshot from 2021-11-10 18-04-25](https://user-images.githubusercontent.com/449228/141207837-c4533732-8684-40d1-805c-733512356509.png)